dlt/common/storages/load_package.py

```diff
@@ -200,6 +201,9 @@ class PackageStorage:
     PACKAGE_COMPLETED_FILE_NAME = (  # completed package marker file, currently only to store data with os.stat
         "package_completed.json"
     )
+    LOAD_PACKAGE_STATE_FILE_NAME = (  # internal state of the load package, will not be synced to the destination
+        "load_package_state.json"
+    )
```

Comment:
should i have one new file for the loadpackage state and keep the package_completed.json as is, or should we have a properly typed state dict that has both infos?